sessions/R-Stan-and-statistical-concepts.qmd

```diff
@@ -28,20 +28,19 @@ The source file of this session is located at `sessions/R-Stan-and-statistical-c
 
 ## Libraries used
 
-In this session we will use the `ggplot2` library for plotting, the `here` library to reference files, for example, the stan model, and the `cmdstanr` library for using stan.
+In this session we will use the `ggplot2` library for plotting, the `nfidd` package to load the stan models, and the `cmdstanr` library for using stan.
 We will also use the `bayesplot` and `posterior` packages to investigate the results of the inference conducted with stan.
 
 ```{r libraries, message = FALSE}
+library("nfidd")
 library("ggplot2")
-library("here")
 library("cmdstanr")
```

Comment:
do we still need these or is this handled by `nfidd` now?

Reply:
We could move it to an imports?

Reply:
My question was more whether we're actually using any `cmdstanr` functions anywhere. No harm in calling the library I guess, but was wondering whether we strictly still need to.